src/main/java/uk/gov/ida/verifylocalmatchingserviceexample/service/Cycle1MatchingService.java

```diff
@@ -54,6 +56,9 @@ public MatchStatusResponseDto matchUser(MatchingServiceRequestDto matchingServic
             return MATCH;
         }
 
+        if(matchingUsers.size() > 1) {
+            return cycle3MatchingService.matchUser(matchingServiceRequest, matchingUsers);
+        }
```

Comment:
Our decision about when to do Cycle 1 matching (when Cycle 0 fails) is in the `MatchingService`. I feel we should put the logic of when to do Cycle 3 matching in there too - it is a bit odd for the `Cycle1MatchingService` to deal with the `Cycle3MatchingService` directly.